api/category.js

```diff
@@ -2,29 +2,36 @@ import axios from "axios";
 import chalk from "chalk";
 import PromptSync from "prompt-sync";
 import { finalData } from "../data/finalData.js";
+import https from "https";
 
 const prompt = PromptSync();
+export const apiBaseURL = "https://api.chucknorris.io/jokes/";
 
 export const listCategories = async () => {
-    let categoryData = {};
+
     try {
-        categoryData = await axios.get("https://api.chucknorris.io/jokes/categories");
+        const categoryData = await axios.get(`${apiBaseURL}categories`, { httpsAgent: new https.Agent({ keepAlive: true }) });
         categoryData.data.forEach((category, i) => {
             console.log(chalk.green("[" + (i + 1) + "]") + " " + chalk.green(category));
```

Comment:
suggestion: try to use template literals for situations like these:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals